rust_pajamax_bench/README.md

```diff
@@ -0,0 +1,11 @@
+Since Pajamax uses thread-mode, it creates a thread for each
+connection, so make sure that
+
+    GRPC_CLIENT_CONNECTIONS >= GRPC_SERVER_CPUS
```

Comment:
Shouldn't it be the other way around?

Reply:
Theoretically speaking,
- if GRPC_CLIENT_CONNECTIONS > GRPC_SERVER_CPUS then
    each CPU will have multiple threads, and switching between threads may affect performance
- else if "<" then
    the CPU will not be fully utilized
- so it's better to set them "=".

In practice, however, since the performance of Pajamax is much higher than that of `ghz`, setting these 2 configs equal would result in a very low CPU utilization rate for Pajamax, less than 10%. Therefore, by setting a higher GRPC_CLIENT_CONNECTIONS, we can measure a higher QPS for Pajamax.

I probably should add all these explanations to the README.

Reply:
I have updated the README.md.